apps/total-typescript/src/video/exercise/github-link.tsx

```diff
@@ -39,7 +40,7 @@ export const GitHubLink: React.FC<{
     module,
   })
 
-  return github.repo && module.github.repo ? (
+  return github.repo && module.github?.repo ? (
```

Comment:
I added `github` the `Module` type, but I think this is a problem with Sanity data where a `github` but be a RESOURCE and not a PROPERTY on the module and then get loaded via trpc